<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Yeelight Bulb Toggle</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Connect device</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>60</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ping</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>currently connected: none</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>device list</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>402</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Bulb settings</string>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>60</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Brightness：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hue</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>on/off</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hue">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>150</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_hue">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>150</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>359</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_ct">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>90</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_ct">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>90</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>1700</number>
     </property>
     <property name="maximum">
      <number>6500</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_bulb_status">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>status: n/a</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_saturation">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>120</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="tracking">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_saturation">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_check">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>30</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>check</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
